scripts/build-layout-scales.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const stripIndent = require('strip-indent');
+const mediaQueries = require('../src/media-queries');
+const allScales = require('../src/scales');
+
+function buildLayoutScales(config) {
+  const layoutScales = typeof config === 'object' ? config : Object.assign(allScales, config);
```

Comment:
Ok, if `config` is going to be an object or `undefined`, I think you can just do `const layoutScales = Object.assign(allScales, config)`.